src/client/api/index.ts

```diff
@@ -56,3 +65,37 @@ type GameSize = { gameSize: number }
 export const fetchGameSize = async (): Promise<number> => {
   return (await request<GameSize>(`/api/gameSize`)).gameSize
 }
+
+let token: string | null = null
+
+const setToken = (newToken: string) => {
+  token = newToken
+}
+
+const getToken = () => {
+  if (!token) {
+    token = localStorage.getItem('token')
+  }
+  return token
+}
```

Comment:
Auch etwas Java-spezifisch 😉

Reply:
ja hast recht :laughing: 
soll keine ausrede sein, aber ich dachte hier "so einfach wie möglich" und im zuge des tickets #70, könnte man das besser umzusetzen da es in dem ticket um das Token-Management geht  :stuck_out_tongue_closed_eyes: